x/liquidity/spec/03_state_transitions.md

```diff
@@ -221,12 +221,26 @@ Variables:
 
 ### Swap Fee Payment
 
-- Swap fees are calculated after the swap price is calculated.
-- Swap fees are proportional to the coins received from matched swap orders.
-
-  - `SwapFee` = `ReceivedMatchedCoin` * `SwapFeeRate`
+To minimize the impact of fee payment process, receive fee half from `OfferCoin`, and the other half with `ExchangedCoin`
```

Comment:
This is just suggestion. How about changing to this?

```suggestion
Rather than taking fee solely from `OfferCoin`, liquidity module is designed to take fee half from `OfferCoin`, and the other half from `ExchangedCoin`. This smoothens out the impact of fee payment process.
```